Pracs/prac_05/emails.py

```diff
@@ -0,0 +1,33 @@
+"""
+CP1404/CP5632 emails
+
+"""
+
+
+def main():
+    """Get email and name from user."""
+    email_to_name = {}
+    email_address = input("Email: ")
+
+    while email_address != "":
+        name = extract_name_from_email(email_address)
+        answer = input(f"Is your name {name}? (Y/n) ").lower()
+
+        if answer != "y" and answer != "":
```

Comment:
Consider adjusting so that the user could enter y/n/yes/no

Reply:
That's a great idea thanks!